### PR TITLE
Recommenders

### PR DESCRIPTION
**Pull request by lilianabs**

### CODE REVIEW

**Review by eliasnema — CHANGES_REQUESTED** [review]
Thanks for adding the content! Overall, looks great.
Small nit: replace all the occurrences of "share your progress in ... in" to "share your progress on .. on".

File: 2022-10-19-recommenders-1.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Project name
 
-* **Goal**: PROJECT_GOAL
-* **Dates**: from DATE_START to DATE_END
+* **Goal**: Learn about the different types of recommendation system algorithms and explore them in Jupyter notebooks. 
+* **Dates**: from 19 October to 2 October
```
Comment (alexeygrigorev) [inline 2022-10-19-recommenders-1.md]: 2 November =)

File: 2022-10-19-recommenders-1.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ If you want to receive reminders about this event, sign up here
 
 ## Technologies 
 
-* Tech 1
-* Tech 2
-* Tech 3
+* Scikit-Learn
```
Comment (alexeygrigorev) [inline 2022-10-19-recommenders-1.md:18]: Do you think scikit learn will be enough for CF recommenders? What about tools like implicit?

File: 2022-10-19-recommenders-1.md [hunk 3]
```diff
@@ -26,48 +26,115 @@ alternatives instead
...
-* Action 2
-* Action 3
-* Create a GitHub project (if day 1)
-* Commit your changes (if day 2+)
-* Share your progress in Slack and in social media
+Found good materials? Create a PR with links!
+
+### Day 4 (22 October, Saturday)
+
+* Learn about Content-based recommendation systems.
```
Comment (alexeygrigorev) [inline 2022-10-19-recommenders-1.md:69]: For content-based we wanted to do another week?

Comment (eliasnema) [reply 2022-10-19-recommenders-1.md:69]: I think we might want to introduce both during the same week and then get deeper on comparison during the 2nd one.

File: 2022-10-19-recommenders-1.md (hunk 1 above)
Comment (lilianabs) [reply 2022-10-19-recommenders-1.md]: Thanks for pointing this out :)

**Comment by lilianabs** [comment]
> 

Thanks for pointing this out :) I've updated the doc.

File: 2022-10-19-recommenders-1.md (hunk 2 above)
Comment (lilianabs) [reply 2022-10-19-recommenders-1.md:18]: I agree, I'll add Implicit and Surprise.

**Comment by alexeygrigorev** [comment]
Thank you Liliana! Just merged it

**Pull request merged**